gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -180,6 +180,7 @@ protected void processMessage(DecodeableKafkaRecord message) {
     String flowGroup = value.getFlowGroup();
     String flowName = value.getFlowName();
     String flowExecutionId = value.getFlowExecutionId();
+    String jobName = value.getJobName();
```

Comment:
will it give error if we do not change the table?
should we have a try/catch so it does not fail when expecting old/new schema? (we may have to rollback)